src/azul/service/avro_pfb.py

```diff
@@ -467,15 +479,17 @@ def _entity_schema_recursive(field_types: FieldTypes,
                 'update_date',
                 'last_modified_date',
             )
-            if path[0] == 'files' and not plural or entity_type in exceptions:
+            if field_name == 'drs_path':
```

Comment:
Extract these field names into class attributes or constants so the codependent sections of code are more easily found.